registration-system/admin/pages_mail.php

```diff
@@ -37,10 +37,13 @@ public function getText() {
         foreach ($this->environment->oconfig['essen'] as $key => $typ) {
             $essen .= '<option value="' . $key . '">' . $typ . '</option>';
         }
-        $maxtage = $this->fahrt->getLenTage();
+        $fahrt_bereich = $this->environment->database->select('fahrten', ['von', 'bis'], ['fahrt_id' => $this->fahrt->getID()]);
```

Comment:
Nicht so elegant... 
Lieber die entsprechende Funktion aus dem Fahrt-Objekt nehmen (bzw neu schreiben)! Ist einheitlicher und wenn sich was ändert, dann nur an einer Stelle. Dafür ist die Klasse ja da.

Reply:
sehe ich mir an

Reply:
nutzt fahrt->getPossibleDates() in der neuen version